src/main/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionController.java

```diff
@@ -117,6 +117,26 @@ public ResponseEntity<String> buildDeletedArtefactSubscribers(@RequestBody Artef
             "Deleted artefact third party subscriber notification request has been accepted");
     }
 
+    @ApiResponses({
```

Comment:
I'm wondering if we need to take a date range in for these (or paginate the data in some way). A huge number os subs eventually could cause this endpoint to timeout if we have a huge number

Reply:
Yeah, that was the plan I had in mind - I was thinking throughout that parameterising could be added later so we could adapt to business reqs. Could either overload with params or require them - what do you reckon?

Reply:
Overload makes sense I reckon!

Happy for this to be adapted / built upon later as the MI reporting grows